adoc/extensions/index.adoc

```diff
@@ -11,4 +11,5 @@ specification, but their design is subject to change.
 // include::sycl_khr_extension_name.adoc[leveloffset=2]
 
 include::sycl_khr_default_context.adoc[leveloffset=2]
+include::sycl_khr_queue_flush.adoc[leveloffset=2]
```

Comment:
Please move this to the end.  New extensions should be added after ones that are already published, so we don't change the section number of the already-published extensions.

Reply:
Fixed with d08148e